terraform/services.tf

```diff
@@ -32,6 +32,10 @@ module "services" {
     "servicenetworking.googleapis.com",
     "secretmanager.googleapis.com",
     "artifactregistry.googleapis.com",
-    "cloudbuild.googleapis.com"
```

Comment:
I see you moved the services from the setup.sh to be managed by terraform. Great!